keyboards/boardsource/3x4/keymaps/default/keymap.c

```diff
@@ -1,9 +1,10 @@
-
 #include QMK_KEYBOARD_H
 
-#define _MAIN 0
-#define _RAISE 1
-#define _LOWER 2
+enum layers {
+    _MAIN=0,
+    _RAISE,
+    _LOWER,
+};
```

Comment:
Mainly an FYI and not required, but using an `enum` statement means the first item is automatically `0`, unless you override it.

This is exactly the same:

```suggestion
enum layers {
    _MAIN,
    _RAISE,
    _LOWER,
};
```